titus-api/src/main/java/com/netflix/titus/api/appscale/store/mixin/AlarmConfigurationMixIn.java

```diff
@@ -31,7 +31,6 @@
             @JsonProperty("region") String region,
             @JsonProperty("actionsEnabled") Optional<Boolean> actionsEnabled,
             @JsonProperty("comparisonOperator") ComparisonOperator comparisonOperator,
-            @JsonProperty("autoScalingGroupName") String autoScalingGroupName,
```

Comment:
Removing this attribute may prevent loading of the existing records from the database. We need to provide a hint to  Jackson deserializer to ignore this value if present.

Reply:
I believe it is the `@JsonIgnoreProperties(ignoreUnknown = true)` annotation.

Reply:
good catch, I thought this was always the default

Reply:
Done. PTAL